article/sections/_4_graph_construction.tex

```diff
@@ -6,64 +6,67 @@ \section{(Initial) Causal Graph Construction}
 The process is similar to how we generated effect modifying variables.
 We iterate through each of the explanatory, mediating, and effect modifying variables, thinking specifically of any variables that both cause the current variable in the iteration and cause the outcome variable(s).
 We call these variables, which cause our outcome and current variables in the iteration, confounding variables.
-We will add each of them to our causal graph.
+As an example, consider a person's attitude towards environmental conservation.
+This attitude may cause both that individual's observed distance to their workplace (another explanatory variable) and that individual's choice of travel mode.
+Both in this example and in general, we should add such confounding variables to our causal graph.
 
-For the last set of variables, we should explicitly consider the role of time, even in research that may be cross-sectional due to the data that is available to us.
+For the last set of variables, we should explicitly consider the role of time, even in research that may be cross-sectional due to the data that is available to us or due to the problem itself.
 In reality, how do we think our system evolves over time?
-If we consider multiple observations of a given decision maker, how do variables observed of that decision maker at time $t$ partially cause future variables important to the context or outcomes for that decision maker at time $t+1$?
+If we consider multiple observations of a given decision maker, how does that decision maker's observed variables at time $t$ partially cause future variables important to the context or outcome(s) for that decision maker at time $t' > t$?
 How do the actions of a decision maker $i$ at time $t$ partially cause the future context or outcomes of a decision maker $j$?
 We should add explicit nodes to our graph, subscripted or denoted by time, to show the cross-time causal relationships in our system.
 
-At this point, we have added all the outcome, explanatory, effect modifying, mediating, confounding, and time-indexed variables that we can think of to our causal graph.
-They are all disconnected nodes, singletons, in the graph.
+At this point, we have added to our causal graph all the outcome, explanatory, effect modifying, mediating, confounding, and time-indexed variables that we believe are relevant for our problem.
+However, they are all disconnected nodes, i.e., singletons in the graph.
 We now focus on pruning nodes from this graph, before drawing our final hypothesized connections.
 In particular, we focus on pruning ``post-outcome'' variables that are not part of the causal graph for future time periods or other observations.
-The reason for this is that conditioning on such post-outcome variables biases one's causal effect estimates.
+The reason for this is that conditioning on such post-outcome variables would bias our causal effect estimates.
```

Comment:
This is interesting. I agree with you totally that conditioning on post-treatment variables is bad in practice if you're interested in the causal effect of a variable, but conditioning on such variables is so common in practice that trying to convince researchers to remove them from there models would cause a lot of resistance, specifically transportation demand models. 

For example, so many variables would be considered "post-treatment" variables if your treatment is a bike lane, e.g., bike ownership, residential choice, cars per household, etc. You will always have those variables in a model, because people are interested in them too. The value of a causal graph here is to keep track of those inter-variable dependencies when coming up with a generative model of the data and estimating structural equations. Even the example of distance -> time/cost could be framed that way if your treatment is "distance" as we discussed. This is where maybe we can rephrase the write-up on those variables to highlight those nuances? Let's discuss!

Reply:
Resolving as I think we discussed the difference was in your thinking post-treatment vs my thinking post-outcome.